src/network/networkd-manager.h

```diff
@@ -38,6 +38,7 @@ struct Manager {
         bool manage_foreign_rules;
 
         Set *dirty_links;
+        Set *new_wlan_links;
```

Comment:
All set or hashmap named with "links" holds `Link` objects.
So, I'd like to rename `new_wlan_links` -> `new_wlan_ifindexes`, `new_wlan_ifindices`, or something like that to indicate that the set holds indexes.